webapp/package.json

```diff
@@ -36,17 +43,16 @@
     "@types/react-router-dom": "5.3.3",
     "@types/shallow-equals": "1.0.0",
     "@types/styled-components": "5.1.25",
-    "@typescript-eslint/eslint-plugin": "5.49.0",
-    "@typescript-eslint/parser": "5.49.0",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@typescript-eslint/parser": "5.62.0",
     "babel-loader": "8.2.3",
     "babel-plugin-formatjs": "10.3.7",
     "babel-plugin-styled-components": "2.0.6",
     "babel-plugin-typescript-to-proptypes": "2.0.0",
     "css-loader": "5.2.6",
-    "eslint": "8.33.0",
+    "eslint": "8.49.0",
     "eslint-import-resolver-webpack": "0.13.1",
     "eslint-plugin-formatjs": "4.6.0",
-    "eslint-plugin-import": "2.27.5",
```

Comment:
I shouldn't have removed this just yet, and yet it's still being pulled into the package-lock, so just leaving as is. Will fix up in the next import order PR (after transcriptions has been rebased and merged into master).